src/m365/todo/commands/task/task-list.ts

```diff
@@ -0,0 +1,120 @@
+import * as chalk from 'chalk';
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import commands from '../../commands';
+import { ToDoTask } from '../../ToDoTask';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  listName?: string;
+  listId?: string;
+}
+
+class ToDoTaskListCommand extends GraphItemsListCommand<ToDoTask> {
+  public get name(): string {
+    return `${commands.TASK_LIST}`;
+  }
+
+  public get description(): string {
+    return 'List tasks from a Microsoft To Do task list';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.listId = typeof args.options.listId !== 'undefined';
+    telemetryProps.listName = typeof args.options.listName !== 'undefined';
+    return telemetryProps;
+  }
+
+  private getListId(args: CommandArgs): Promise<string> {
+    if (args.options.listId) {
+      return Promise.resolve(args.options.listId);
+    }
+
+    const requestOptions: any = {
+      url: `${this.resource}/v1.0/me/todo/lists?$filter=displayName eq '${encodeURIComponent(args.options.listName as string)}'`,
+      headers: {
+        accept: 'application/json;odata.metadata=none'
+      },
+      responseType: 'json'
+    };
+
+    return request.get<{ value: [{ id: string }] }>(requestOptions)
+      .then(response => {
+        const listItem: { id: string } | undefined = response.value[0];
+
+        if (!listItem) {
+          return Promise.reject(`The specified task list does not exist`);
+        }
+
+        return Promise.resolve(listItem.id);
+      });
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+    this
+      .getListId(args)
+      .then((listId: string): Promise<any> => {
+        const endpoint: string = `${this.resource}/v1.0/me/todo/lists/${listId}/tasks`;
+        return this.getAllItems(endpoint, logger, true)
+      })
+      .then((): void => {
+        if (args.options.output === 'json') {
+          logger.log(this.items);
+        }
+        else {
+          logger.log(this.items.map(m => {
+            return {
+              title: m.title,
+              importance: m.importance,
```

Comment:
`id` and `status` makes more sense to me to print out by default than `importance` and `isReminderOn`. What do you think?